README.md

```diff
@@ -2,28 +2,22 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.1.3.
 
-## Development server
-
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
-
-## Code scaffolding
+You can view the latest project version visiting: https://johngtka.github.io/BioMagnetic-app/about
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+or
 
-## Build
+visit [This](http://book-with-me.free.nf/home)
 
-To make a build checkout master branch and make git pull.
+This application is for the clinic's customer service to ensure the highest quality of service.
 
-Run `npm run build` to build the project. This process will trigger 3 processes:
+# Appointments
 
--   prebuild where the varsion number will be adjusted, new tag will be created
--   build process -> ng build
--   post build where changes will be push to the branch and dist codebase will be push to github-pages.
+To create appointment let's go here [FakeCompany](https://fakecompany125.simplybook.it/v2/)
 
-The build artifacts will be stored in the `dist/sana` directory.
+Here you can generate test visits using this service
 
-You can view the latest project version visiting: https://johngtka.github.io/BioMagnetic-app/about
+# Backend
 
-## Running unit tests
+to connect with api, use postman with [API](https://busy-tan-whale-tam.cyclic.app)
```

Comment:
Backend is hosted on `https://www.cyclic.sh/` and it's a Mongo Express Angular (frontend) Node stack